### PR TITLE
fix: do not fail when dist has not changed

### PR DESCRIPTION
**Pull request by abulte**

### CODE REVIEW

File: .github/workflows/update-universes.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ jobs:
         git add logs/run_log_demo.txt
         git commit -m "Update daily run log for demo"
         git add dist/organizations-demo.json
-        git commit -m "Update organizations JSON for demo"
+        git commit -m "Update organizations JSON for demo" || true
```
Comment (streino) [inline .github/workflows/update-universes.yml:41]: Any failure will be ignored, including actual commit issues (not sure what they'd be, but you never know...)

```suggestion
        git diff-index --cached --quiet HEAD || git commit -m "Update organizations JSON for demo"
```

From https://stackoverflow.com/a/3879077:
> The safer approach seems to be to run git add on the file spec first and then use git diff-index to see if anything got added to index before running git commit.
> ```
> git add ${file_args} && \
> git diff-index --cached --quiet HEAD || git commit -m '${commit_msg}'
> ```

Comment (abulte) [reply .github/workflows/update-universes.yml]: Right! I tried that this morning, but on the wrong repo with an ignored dist, so I could not get a non zero status code 😬 Works well on the right repo.

File: .github/workflows/update-universes.yml [hunk 2]
```diff
@@ -78,7 +78,7 @@ jobs:
         git add logs/run_log_prod.txt
         git commit -m "Update daily run log for prod"
         git add dist/organizations-prod.json
-        git commit -m "Update organizations JSON for prod"
+        git commit -m "Update organizations JSON for prod" || true
```
Comment (streino) [inline .github/workflows/update-universes.yml:81]:
```suggestion
        git diff-index --cached --quiet HEAD || git commit -m "Update organizations JSON for prod"
```

Comment (abulte) [reply .github/workflows/update-universes.yml:81]: 🤦

**Review by streino — APPROVED** [review]

**Pull request merged**